environment.yml

```diff
@@ -49,6 +49,7 @@ dependencies:
       - sphinxcontrib-svg2pdfconverter>=1.1.0
       - sphinxcontrib-video>=0.2.1
       - pikepdf
+      - PyStemmer
```

Comment:
PyStemmer is available as package on conda-forge under the name `pystemmer`.  Please use that instead of the pip package.

Reply:
Got you, I just change that